access_requests.go

```diff
@@ -0,0 +1,240 @@
+package gitlab
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+)
+
+// AccessRequest represents a access request for a group or project.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/access_requests.html
+// TODO: try to find existed structure for that
+type AccessRequest struct {
+	ID        int        `json:"id"`
+	Username  string     `json:"username"`
+	Name      string     `json:"name"`
+	State     string     `json:"state"`
+	CreatedAt *time.Time `json:"created_at"`
+	// RequestedAt represents a access request requested time. Can be empty, when used
+	// ApproveProjectAccessRequest() or ApproveGroupAccessRequest().
+	RequestedAt *time.Time `json:"requested_at"`
+	// AccessLevel represents a user access level. Can be empty, when used ListProjectAccessRequests(),
+	// ListGroupAccessRequests(), RequestProjectAccess(), RequestGroupAccess().
+	AccessLevel *AccessLevelValue `json:"access_level,omitempty"`
+}
```

Comment:
I prefer to not litter the struct with these kinds of comments. People using the API would know this (or can find it in the docs).